service/src/test/java/com.codeforcommunity/JooqMock.java

```diff
@@ -198,11 +195,43 @@ int getCallCount() {
     }
   }
 
-  /** Constructor for JooqMock. 'UNKNOWN' and 'DROP/CREATE' operations are added by default. */
+  /**
+   * Represents an operation type in SQL. For all explicitly unsupported types you want to use, the
+   * {@link OperationType#UNKNOWN} type acts as a catchall.
+   */
+  public enum OperationType {
```

Comment:
I created this enum and deprecated methods that used strings to determine the operation to perform.